README.md

```diff
@@ -1,68 +1,79 @@
-<p align="center">
-<img src="/logo.png" alt="butler" style="max-width:100%;">
-</p>
+![butler](/logo.png)
 
-Butler is an automatation tool to scaffold new projects in only a few seconds. We provide a powerful interactive cli.
-When you create a project template you can create a [`Survey`](/docs/templateSurveys.md). Surveys are used to collect informations from the users to generate individual templates. Beside templating we also plan to integrate common tasks of popular Project Managents Tools like Jira, Confluence in Butler.
+Butler is an automatation tool to scaffold new projects in only a few seconds.
+We provide a powerful interactive cli. When you create a project template you
+can create a [`Survey`](/docs/templateSurveys.md). Surveys are used to collect
+informations from the users to generate individual templates. Beside
+templating we also plan to integrate common tasks of popular Project Managents
+Tools like Jira, Confluence in Butler.
 
 > Bootstraping projects should be fun!
 
-## Features
+# Features
+
 - Template Surveys
 - Conditional files and folders
 - After hooks for post-processing
 - Auto Update
 - Distributed configs
 
-## Principles
+# Principles
+
 - Project Templates are simple git repositories
-- Everything is a template you don't have to deal with `/template` directories or `.tmpl` files
+- Everything is a template you don't have to deal with `/template` directories
+  or `.tmpl` files
 - Required informations are asked during the bootstrapping process
 
-## Usage
+# Usage
 
 1. [Download here](https://github.com/netzkern/butler/releases)
 2. Install in `PATH`
 3. Run `butler`
 
-## Documentation
+# Documentation
 
-* <a href="/docs/config.md"><code><b>Config</b></code></a>
-* <a href="/docs/templateSurveys.md"><code><b>Template Surveys</b></code></a>
-* <a href="/docs/templateSyntax.md"><code><b>Template Syntax</b></code></a>
-* <a href="/docs/gitHooks.md"><code><b>Git Hooks</b></code></a>
-* <a href="/docs/debugging.md"><code><b>Debugging</b></code></a>
-* <a href="#commands"><code><b>Commands</b></code></a>
+- [`**Config**`](/docs/config.md)
```

Comment:
This looks wrong.

Reply:
the syntax is correct here

Reply:
No, look at the preview the text should be bold.